components/shared/Footer.js

```diff
@@ -1,21 +1,76 @@
-import React from "react";
 import Container from "react-bootstrap/Container";
 import Row from "react-bootstrap/Row";
 import Col from "react-bootstrap/Col";
 import Image from "react-bootstrap/Image";
+import { FaRegEnvelope } from "react-icons/fa";
+import IconLabel from "./IconLabel";
 
-export default function Footer(props) {
+export const LINKS = [
+  { label: "About", href: "https://austinmobility.io" },
```

Comment:
would you update this `href` to `/about`, so that it directs us to https://austinmobility.io/about ?